irepela/homework_7/retrying.py

```diff
@@ -0,0 +1,26 @@
+"""
+Create decorator, which will retry inner function multiple times until it passes
+"""
+
+
+import random
+
+
+def retry(func):
+
+    def wrapper(*args, **kwargs):
+        while True:
+            try:
+                return func(*args, **kwargs)
+            except Exception as e:
+                print(e)
+
+    return wrapper
```

Comment:
also try to prepare parametrized decorator, which expects retry count, if not defined - unlimited retries